types/readable-stream/package.json

```
@@ -20,6 +20,10 @@
         {
             "name": "markdreyer",
             "githubUsername": "markdreyer"
+        },
+        {
+            "name": "mcollina",
```

This can be a full name:

```suggestion
            "name": "Matteo Collina",
```

This package is actually an outlier for not having this format.